CREDITS

```diff
@@ -530,7 +536,7 @@ lib/base64.c:
 ==============================================================================
 
 include/heap.h,
-lib/heap.cc:
+lib/heap.c:
```

Comment:
I tried to check all filenames at the top of CREDIT entries. If I missed any, please adjust this PR as needed.